parsing/intermediate/dialects/dialect1/parserImpls.ftl

```diff
@@ -2780,6 +2780,21 @@ SqlNode TableRefOrJoinClause() :
     { return e; }
 }
 
+void CaretNegation(ExprContext exprContext, List<Object> list) :
+{
+    final SqlOperator op;
+}
+{
+    <CARET>
+    Expression2b(exprContext, list)
+    <EQ> {
+        op = SqlStdOperatorTable.NOT_EQUALS;
```

Comment:
nit: Can we just omit the temporary variable altogether and pass SqlStdOperatorTable.NOT_EQUALS directly into the ToTreeListItem() call?

Reply:
Yeah I'll update that